Probesys/bileto
# dev: Remove the symfony/phpunit-bridge dependency

## Related issue(s)



https://github.com/Probesys/bileto/issues/674

## How to test manually



- Run the tests suite
- Verify that it doesn't display "THE ERROR HANDLER HAS CHANGED!" at the end

## Checklist
